themes/bootstrap5/templates/error/index.phtml

```diff
@@ -7,12 +7,7 @@
 <div class="alert alert-danger">
   <p><?=$this->transEsc('An error has occurred')?></p>
   <p><?=$this->transEsc($this->message)?></p>
-  <p>
-    <?=$this->transEsc('Please contact the Library Reference Department for assistance')?>
-    <br>
-    <?php $supportEmail = $this->escapeHtmlAttr($this->systemEmail()); ?>
-    <a href="mailto:<?=$supportEmail?>"><?=$supportEmail?></a>
-  </p>
+  <p><?=$this->render('error/contact.phtml')?></p>
```

Comment:
Now that error/contact.phtml contains complete `<p>` tags of its own, I think you can remove the wrapping paragraph in all the templates that include it.

Reply:
Right, of course! Now fixed.